website/docs/en/config/experiments.mdx

```diff
@@ -150,15 +150,15 @@ Starting from v1.4.0, Rspack enables incremental builds for all phases by defaul
 
 ## experiments.parallelCodeSplitting
 
-<ApiMeta addedVersion="1.2.0" />
+<ApiMeta deprecatedVersion="1.6.2" />
```

Comment:
We can move the deprecated options to the bottom of the page

Reply:
See: https://rspack.rs/config/experiments#experimentstoplevelawait

Reply:
Done